modules/ps_searchbar/ps_searchbar.tpl

```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 
-<div id="_desktop_search" class="order-2 ms-auto">
+<div id="_desktop_search" class="order-2 ms-auto col-auto px-1 hidden-on-mobile">
```

Comment:
We wanted to get rid of these `hidden-on-*` classes but seems like we forgot to remove them, to rely more on bootstrap classes as you specified in the description